src/sass/components/_path.scss

```diff
@@ -27,7 +26,7 @@ $color-background-path-current: $brand-background-primary;
 
     &:before,
     &:after {
-      background: $color-background-path-complete;
+      background: inherit;
```

Comment:
@mirisuzanne I added inherit when possible.

Reply:
@stacyk since you added it here, I don't think you should need to restate it inside `&:hover`, right?

Reply:
@mirisuzanne when I don't have it there, then the LDS styles  for the `.foo:hover:before` and `...:after`are used since they specify everything explicitly.

![Screen Shot 2021-05-21 at 9 57 41 PM](https://user-images.githubusercontent.com/1581694/119212705-b843dc00-ba7f-11eb-91cf-1234a0fefa82.jpg)